src/main/asciidoc/_chapters/ops_mgt.adoc

```diff
@@ -3191,40 +3208,38 @@ full implications and have a sufficient background in managing HBase clusters.
 It was developed by Yahoo! and they run it at scale on their large grid cluster.
 See link:http://www.slideshare.net/HBaseCon/keynote-apache-hbase-at-yahoo-scale[HBase at Yahoo! Scale].
 
-RSGroups are defined and managed with shell commands. The shell drives a
-Coprocessor Endpoint whose API is marked private given this is an evolving
-feature; the Coprocessor API is not for public consumption.
+RSGroups can be defined and managed with both admin methods and shell commands.
```

Comment:
Are these RSGroup related doc changes made on purpose or the cherry-pick introduced them?

Reply:
That was introduced automatically by the cherry-pick. Need to revert it.